package.json

```diff
@@ -1,16 +1,61 @@
 {
+  "name": "@tableland/local",
+  "version": "0.0.0-pre10",
+  "description": "Tooling to start a sandboxed Tableland network.",
+  "repository": "https://github.com/tablelandnetwork/local-tableland",
+  "license": "MIT",
+  "publishConfig": {
+    "access": "public"
+  },
+  "keywords": [
+    "tableland",
+    "sql",
+    "ethereum"
+  ],
+  "engines": {
+    "node": ">=12.0.0"
+  },
+  "main": "./bin/cjs/main.js",
```

Comment:
I was looking at how other libs were doing npx commands and it seemed like it was common to use bin when there is a "bin" property in package.json.  Should we change to dist?

Reply:
Naw its fine. I usually see bin for the binary and dist for the rest... but since this is entirely a cli project, it seems totally fine to have it all together

Reply:
That's a good point.  Future TODO: